src/terra-dev-site/doc/Configuration.b/wdio.conf.d.doc.md

```diff
@@ -0,0 +1,4 @@
+# Wdio Config
+Terra dev site's wdio config adds the `data-terra-site-content` selector and a before hook to automatically refresh the page.
```

Comment:
Terra-dev-site's wdio config **extends the [terra-toolkit's default wdio config](https://github.com/cerner/terra-toolkit/blob/master/config/wdio/wdio.conf.js) by adding** ~add~